app/code/Magento/Checkout/Model/Session.php

```diff
@@ -41,6 +41,15 @@ class Session extends \Magento\Framework\Session\SessionManager
      */
     protected $_loadInactive = false;
 
+    /**
+     * A flag to track when the quote is being loaded and attached to the session object.
+     *
+     * Used in trigger_recollect infinite loop detection.
+     *
+     * @var bool
+     */
+    protected $_isLoading = false;
```

Comment:
Please, avoid underscore in var name and make variable private since it used in the current class only.

Reply:
@sidolov actioned in 746add61c4a0718fbe7b842a350734850de751ca.

To me this is inconsistent with the rest of the variables in the class but okay.